components/TradingCardGrid.jsx

```diff
@@ -34,27 +36,24 @@ export default function TradingCardGrid({ teamMembers }) {
   };
 
   useEffect(() => {
-    const filterTeamMembers = (allTeamMembers) => (
-      allTeamMembers.filter((member) => {
-        const hoursIdx = getForecastedHoursIdx(weekOffset);
-        const hasHoursAvailable = member.weeklyCapacity - member.forecastedHours[hoursIdx] > 0;
-
-        if (!hasHoursAvailable && availability === availabilityOptions.AVAILABLE) return false;
-        if (hasHoursAvailable && availability === availabilityOptions.UNAVAILABLE) return false;
-
-        if (roles.length && !roles.includes(member.role)) return false;
+    // eslint-disable-next-line max-len
+    const isAvailable = (member) => member.weeklyCapacity - member.forecastedHours > AVAILABILITY_CUTOFF;
+    // eslint-disable-next-line max-len
+    const hrsFilter = (member) => availability === availabilityOptions.AVAILABLE === isAvailable(member);
```

Comment:
I added `eslint-disable-next-line max-len` to the above two lines because the rule would require both these lines to be split into two and I strongly feel that they are easier to understand if they are not split. Additionally both lines are only 5 char over the rule limit.